src/main/java/edu/harvard/iq/dataverse/externaltools/ExternalToolHandler.java

```diff
@@ -21,7 +25,7 @@
  */
 public class ExternalToolHandler {
 
-    public static boolean hardCodedKludgeForDataExplorer = false;
+    public static boolean hardCodedKludgeForDataExplorer = true;
```

Comment:
I gave this the name `hardCodedKludgeForDataExplorer` to make it somewhat clear that we'd like to remove this field entirely. Removing it will purposefully cause us to look at other parts of the code.